rails-snow-queen/app/views/quote_mailer/new_quote.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Snow Queen service request #<%= @quote.id %></h1>
+
+<h2>Hi <%= @quote.first_name %></h1>
+
+<p>
+  Thank you for choosing Snow Queen to fulfill your winter snow clearing and salting needs! <br>
+  For your reference, here are the details of the service you requested on <%= @quote.created_at %>:
+</p>
+<dl>
+  <dt>Address:</dt>
+  <dd><%= @quote.address %></dd>
+  <dt>Area to be serviced:</dt>
+  <dd><%= @quote.area.toFixed(0) %>sq ft</dd>
+  <dt>Comments:</dt>
+  <dd><%= @quote.comments %></dd>
+  <dt>Total due:</dt>
+  <dd>$<%= @quote.total.toFixed(2) %></dd>
+</dl>
+
```

Comment:
Not sure if leaving as a "nice to have" but without a static map (or at least a back end interface), the service provider doesn't know which area to plow/salt